### PR TITLE
update Makefile

### PR DESCRIPTION
**Pull request by Mitchina**

### CODE REVIEW

File: {{cookiecutter.project_dirname}}/Makefile [hunk 1]
```diff
@@ -26,22 +26,21 @@ createsuperuser:  ## Django createsuperuser
...
-	python3 -m manage dumpdata auth.Group --natural-foreign --natural-primary -o fixtures/auth_groups.json
+	python3 -m manage dumpdata auth.Group --natural-foreign --natural-primary --output fixtures/auth_groups.json
 
 .PHONY: fix
 fix:  ## Fix code formatting, linting and sorting imports
 	python3 -m black .
-	python3 -m isort .
-	python3 -m flake8
-	python3 -m mypy .
+	ruff --fix .
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/Makefile:34]: remove ruff and restore older command

File: {{cookiecutter.project_dirname}}/Makefile [hunk 2]
```diff
@@ -53,31 +52,36 @@ local: pip_update  ## Install local requirements and dependencies
...
+	python3 -m piptools compile --generate-hashes --no-header --quiet --resolver=backtracking --upgrade --output-file requirements/base.txt requirements/base.in
+	python3 -m piptools compile --generate-hashes --no-header --quiet --resolver=backtracking --upgrade --output-file requirements/common.txt requirements/common.in
+	python3 -m piptools compile --generate-hashes --no-header --quiet --resolver=backtracking --upgrade --output-file requirements/local.txt requirements/local.in
+	python3 -m piptools compile --generate-hashes --no-header --quiet --resolver=backtracking --upgrade --output-file requirements/remote.txt requirements/remote.in
+	python3 -m piptools compile --generate-hashes --no-header --quiet --resolver=backtracking --upgrade --output-file requirements/test.txt requirements/test.in
 
 .PHONY: pip_update
 pip_update:  ## Update requirements and dependencies
-	python3 -m pip install -q -U pip~=22.0.0 pip-tools~=6.6.0 setuptools~=62.1.0 wheel~=0.37.0
+	python3 -m pip install --quiet --upgrade pip~=22.3.0 pip-tools~=6.12.0 setuptools~=65.6.0 wheel~=0.38.0
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/Makefile:84]: Restore older versioins for now

File: {{cookiecutter.project_dirname}}/Makefile [hunk 3]
```diff
@@ -99,6 +103,30 @@ remote: pip_update  ## Install remote requirements and dependencies
 report:  ## Run coverage report
 	./scripts/report.sh
 
+.PHONY: runserver
+runserver:  ## Django run
+	python3 -m manage runserver 0:8000
+
+.PHONY: scan_docker_image
+scan_docker_image:  ## Scan docker image vulnerabilities
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/Makefile:111]: remove this one, will be added in separate PR

File: {{cookiecutter.project_dirname}}/Makefile [hunk 4]
```diff
@@ -99,6 +103,30 @@ remote: pip_update  ## Install remote requirements and dependencies
...
+	--no-progress \
+	--cache-dir .trivycache/ \
+	--format template --template "@contrib/html.tpl" \
+	--output "/app/.trivy/report.html" \
+	--security-checks vuln \
+	--severity CRITICAL,HIGH,MEDIUM,LOW \
+	{{ cookiecutter.project_slug }}_backend:latest
+
+.PHONY: scan_terraform
+scan_terraform:  ## Scan terraform vulnerabilities
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/Makefile:127]: remove this one, will be added in separate PR

File: {{cookiecutter.project_dirname}}/Makefile (hunk 1 above)
Comment (Mitchina) [reply {{cookiecutter.project_dirname}}/Makefile:34]: okk

File: {{cookiecutter.project_dirname}}/Makefile (hunk 2 above)
Comment (Mitchina) [reply {{cookiecutter.project_dirname}}/Makefile:84]: okk

File: {{cookiecutter.project_dirname}}/Makefile (hunk 3 above)
Comment (Mitchina) [reply {{cookiecutter.project_dirname}}/Makefile:111]: okk

File: {{cookiecutter.project_dirname}}/Makefile (hunk 4 above)
Comment (Mitchina) [reply {{cookiecutter.project_dirname}}/Makefile:127]: okk

**Review by niccolomineo — APPROVED** [review]

**Review by pauloxnet — APPROVED** [review]
